### PR TITLE
feat-avatar styling

### PR DESCRIPTION
**Pull request by ImrozNezami**
different styles of avatar added
avatar of different size
avatar with background color
Screenshot (211)

### CODE REVIEW

**Comment by ImrozNezami** [comment]
Screenshot (211)

different styles of avatar added

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+.av-md {
+  width: 7rem;
+  height: 7rem;
+}
+.av-lg {
+  width: 10rem;
+  height: 10rem;
+}
+.circle {
```
Comment (tanishq20) [inline avatar.css]: You can shift this class to the utility folder so that it can be used in some other file also if required.

**Review by tanishq20 — COMMENTED** [review]
Commented small change.
Rest everything is fine good to merge.

**Review by ttahm3d — COMMENTED** [review]
If the entire changes have not been committed, please commit again.

File: avatar.css (hunk 1 above)
Comment (ImrozNezami) [reply avatar.css]: OK will do it

**Pull request merged**